std/algorithm/iteration.d

```diff
@@ -4874,13 +4880,14 @@ private struct UniqResult(alias pred, Range)
     }
 }
 
-// permutations
+/// permutations
 struct Permutations(Range)
     if (isRandomAccessRange!Range && hasLength!Range)
 {
-    size_t[] indices, state;
-    Range r;
+    private size_t[] indices, state;
```

Comment:
You need to deprecate these, perferably in a different PR. See https://github.com/dlang/phobos/pull/3950/files